admin_pages/general_settings/templates/country_details_settings.template.php

```diff
@@ -26,7 +26,7 @@ class="button-primary save"
             <input
                 type="hidden"
                 id="country-currency-setting-disabled-input"
-                value="<?php echo $CNT_cur_disabled ? 'true' : 'false'; ?>"
+                value="<?php echo ($CNT_cur_disabled ? 'true' : 'false'); ?>"
```

Comment:
@tn3rb What is your opinion about using this code style. I know the WP guys searches for `echo $` to find non escaped variables. Based on my previous experience they do not go into the details and as soon as they see that something is matched with their search they think that we did not escaped all the variables.

Using this syntax we can remove these stuff from their search.

Reply:
lolz... good demonstration of how useless their security reviews are 😆 

i personally don't like the extra brackets and they may conflict with our linting rules, which are set to try and follow the [PHP PSR 12](https://www.php-fig.org/psr/psr-12/) style guidelines.

That said, it's fine in this case as a quick fix. Would be better imho to refactor things so that we don't have to convert that boolean to string `"true"` or `"false"` but that could end up being a rabbit hole we don't want to go down, so best to go with the quick fix for now.

So ya, that's fine with me